docs/product-assembly_dm-on-openshift/src/main/asciidoc/imagestreams-file-install-proc.adoc

```diff
@@ -1,26 +1,49 @@
 [id='imagestreams-file-install-proc']
 = Ensuring the availability of image streams 
 
-You must ensure that the image streams that are required for the deployment are available in your OpenShift environment. Some versions of the OpenShift environment include the necessary image streams. You must check if they are available. If they are not available, you must download and install the `rhdm70-image-streams.yaml` file.
+You must ensure that the image streams that are required for the deployment are available in your OpenShift environment. Some versions of the OpenShift environment include the necessary image streams. You must check if they are available. If they are not available, you must download and install the
+ifdef::DM[`rhdm70-image-streams.yaml`]
+ifdef::PAM[`rhpam70-image-streams.yaml`]
+file.
 
 .Procedure
 . Run the following commands:
 +
+ifdef::DM[]
 [subs="verbatim,macros"]
 ----
 $ oc get imagestreamtag -n openshift | grep rhdm70-decisioncentral-openshift
 $ oc get imagestreamtag -n openshift | grep rhdm70-kieserver-openshift
 ----
+endif::DM[]
+ifdef::PAM[]
+[subs="verbatim,macros"]
+----
+$ oc get imagestreamtag -n openshift | grep rhpam70-businesscentral
+$ oc get imagestreamtag -n openshift | grep rhpam70-kieserver
+----
+endif::PAM[]
 +
 If the outputs of both commands are not empty, the required image streams are available and no further action is required.
-
-. If the output of one or both of the commands is empty, download the `rhdm70-image-streams.yaml` file from {URL_BASE_GITHUB_DM}. Complete one of the following actions:
++
+. If the output of one or both of the commands is empty, download the
+ifdef::DM[`rhdm70-image-streams.yaml`]
+ifdef::PAM[`rhpam70-image-streams.yaml`]
+file from {URL_BASE_GITHUB_DM}. Complete one of the following actions:
 +
 ** Run the following command:
 +
+ifdef::DM[]
 [subs="verbatim,macros"]
 ----
 $ oc create -f rhdm70-image-streams.yaml
 ----
+endif::DM[]
+ifdef::PAM[]
+[subs="verbatim,macros"]
+----
+$ oc create -f rhpam70-image-streams.yaml
+----
+endif::PAM[]
 +
 ** Using the OpenShift Web UI, select *Add to Project > Import YAML / JSON*, then choose the file or paste its contents.
```

Comment:
Our style for UI series is ->, so *Add to Project -> Import YAML ..." I'd search for all text > text and if it's a UI action series, replace with ->. Trivial, but just letting you know for future ref.